zeppelin-web/karma.conf.js

```diff
@@ -67,6 +67,7 @@ module.exports = function(config) {
       'bower_components/ace-builds/src-noconflict/keybinding-emacs.js',
       'bower_components/ace-builds/src-noconflict/ext-language_tools.js',
       'bower_components/ace-builds/src-noconflict/theme-chrome.js',
+      'bower_components/ace-builds/src-noconflict/mode-javascript.js',
```

Comment:
I saw similar changes in another PR #4424 so you guys need to decide who will add it. :-) @HYEEWON

Reply:
It doesn't matter who adds it. But, I added it first, so I want to add it. Is it okay?

Reply:
I also think the file was generated when building on window. But I don't know why the file is auto-generated. First of all, I pushed another commit with mode-javascript.js removed.

Reply:
Thank you :)

Reply:
Thank you for taking this issue.